sdk/core/azure-core/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Fixed and issue where WinHttp transport was not closing correctly in case of request timeout.
```

Comment:
```suggestion
- [[5244](https://github.com/Azure/azure-sdk-for-cpp/issues/5244)] Fixed an issue where WinHttp transport was not closing correctly in case of request timeout.
```

Reply:
Formatting nit around the changelog.